src/Order.mo

```diff
@@ -42,5 +42,17 @@ module {
       case _ { false }
     }
   };
-
+  public func lteToOrder<X>(isLessThanOrEqual : (X, X) -> Bool) : (X, X) -> Order {
```

Comment:
I wonder if there is a better name for this...

totalOrderToComparer

or 

lessThanOrEqualToCompare

or 
compareBy